File: LinguagemC/media.c
/*
Algoritmos Resolvidos em C
Prof.José Augusto Cintra - http://josecintra.com/blog

Enunciado: Cálculo da média entre 2 números informados pelo usuário

*/ 

#include <stdio.h>

int main()
{
  //Declaração de variáveis
  float nota1, nota2, media;
  //Entrada de dados
  printf ("Digite primeira nota: ");
  scanf ("%f" ,&nota1);
  printf ("Digite segunda nota: ");
  scanf ("%f" ,&nota2);
  //Processamento
  media = (nota1 + nota2) / 2;
  // Saída de dados
  printf ("A média entre %f e %f é igual a %f", nota1, nota2, media);
}

File: LinguagemGO/media.go
/*
Algoritmos Resolvidos em GOLANG

Prof.José Augusto Cintra 
http://josecintra.com/blog
https://github.com/JoseCintra/AlgoritmosResolvidos

Enunciado: Leia duas notas notas e calcule a média aritmética

*/
package main

import "fmt"

func main() {
	var nota1, nota2, media float64
	fmt.Print("Informe a nota 1: ")
	fmt.Scan(&nota1)
	fmt.Print("Informe a nota 2: ")
	fmt.Scan(&nota2)
	media = (nota1 + nota2) / 2.0
	fmt.Printf("A média entre %f e %f é %f", nota1, nota2, media)
}
